backend/app/routers/session.py

```diff
@@ -46,8 +48,21 @@ async def delete_session(
     await session_service.delete_session(db_session, session_id, current_user.id)
     return session_id
 
+
 @router.get("/{session_id}/ranking", response_model=list[UserSessionRankingOut])
 async def get_session_ranking(
     db_session: Annotated[AsyncSession, Depends(get_session)], session_id: str
 ) -> list[UserSessionRankingOut]:
     return await session_service.get_session_ranking(db_session, session_id)
+
+
+@router.post("/{session_id}/ranking/{user_name}/submit", response_model=QuizCheckOut)
```

Comment:
I'm not sure whether having a user name in the URL is a good idea 🤔

Reply:
Renaming the endpoint as "/{session_id}/solution/submit" and placing the user_name in the request body will be a better option?

Reply:
IMO, yes